docs/documentation/guides/event-filter.mdx

```diff
@@ -0,0 +1,503 @@
+---
+title: "Event Filters"
+description: "A guide to using event filters to create more precise and tailored job triggers."
+---
+
+## What are Event Filters?
+
+Event filters provide a powerful way to define specific conditions that determine when a job should be triggered based on incoming events. Filters allow you to narrow down event processing to only those events that match certain criteria, ensuring that your jobs are executed in a targeted manner.
+
+we will cover the various types of event filters and how they can be used to create more precise triggers.
+
+### Event Filter Types
```

Comment:
I think this whole doc should start with pretty much the exact content from here:

https://trigger.dev/docs/documentation/concepts/triggers/events#event-filters

Basically just take that content and then go straight to String Filters